docs/src/HowToGuides/Numerics/ODESolvers/Timestepping.md

```diff
@@ -90,8 +90,8 @@ For example, consider the following:
 ```julia
 export MyIMEXMethod
 
-using ..LinearSolvers
-const LS = LinearSolvers
+using ..SystemSolvers
+const LS = SystemSolvers
```

Comment:
maybe do `LS` -> `SS`, just so future people don't get confused, but not entirely necessary

Reply:
We can actually just delete this since everything is in one module.